frontend/public/components/nav/admin-nav.tsx

```diff
@@ -85,193 +87,192 @@ const MonitoringNavSection_ = ({ canAccess }) => {
 };
 const MonitoringNavSection = connect(monitoringNavSectionStateToProps)(MonitoringNavSection_);
 
-const AdminNav = () => (
-  <>
-    <NavSection title="Home">
-      <HrefLink
-        href="/dashboards"
-        activePath="/dashboards/"
-        name="Overview"
-        required={[FLAGS.CAN_GET_NS, FLAGS.OPENSHIFT]}
-      />
-      <ResourceClusterLink resource="projects" name="Projects" required={FLAGS.OPENSHIFT} />
-      <HrefLink href="/search" name="Search" startsWith={searchStartsWith} />
-      <HrefLink href="/api-explorer" name="Explore" startsWith={apiExplorerStartsWith} />
-      <ResourceNSLink resource="events" name="Events" />
-    </NavSection>
+const AdminNav = () => {
+  const lastNamespace = localStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
+  // Machines, Machine Sets, Machine Autoscalers, and Machine Health Check are only available in certain namespaces, so we can't use ResourceNSLink. This allows for behavior consistent with other resources when "all projects" is selected.
```

Comment:
This is a long line. I'm surprised prettier didn't wrap it. Do you mind wrapping it?

I'd say something like

> In OpenShift, machines are created in the `openshift-machine-api` namespace. Switch to that namespace so the list isn't empty. If "all projects" was last selected, however, use "all projects" instead.

Reply:
All set.